Add description of new longhorn storage value (#876)
harvester/docs: docs/install/harvester-configuration.md

        guaranteedInstanceManagerCPU: 6
```

### `install.harvester.longhorn.default_settings.storageReservedPercentageForDefaultDisk`

**Versions**: v1.7.0 and later

**Definition**: Percentage of disk space that will not be allocated to the default disk on each new Longhorn node.

> Harvester creates a separate partition on the boot disk for the Longhorn default disk. Reserving space on the default root disk for longhorn is redundant in that case.

Reviewer: ```suggestion
Harvester creates a dedicated partition on the boot disk for the Longhorn default disk, so you do not need to reserve any space for it on the root disk.
```
